examples/learn_search_space/__init__.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.transfer` -- Coordination of transfer optimization of learning search space
+======================================================================
+
+.. module:: transfer
+   :platform: Unix
+   :synopsis: Executes transfer optimization of learning search space.
+
```

Comment:
There should be a citation to the paper here.

Reply:
fixed.